kolibri/core/assets/src/api-resource.js

```diff
@@ -186,7 +186,7 @@ class Model {
             this.resource.removeModel(this);
```

Comment:
But, you should probably change the reject message, as we can't pass in a model id now, yes?